src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -554,17 +554,20 @@ internal void StartClientTimeout()
 
         private void CheckClientTimeout()
         {
-            // If it's been too long since we've heard from the client, then close this
-            if (DateTime.UtcNow.Ticks - Volatile.Read(ref _lastReceivedTimeStamp) > _clientTimeoutInterval)
+            if (!Debugger.IsAttached)
```

Comment:
I'm a huge fan of reducing nesting by exiting early. So in this case:

```C#
if (Debugger.IsAttached)
{
    return;
}
```

Reply:
Done!